pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -262,7 +262,7 @@ def _get_output_shape_from_returnn(self,
 
     out_torch_shape = [input.shape[perm[i]] for i in range(ndim)]
     out_returnn_axis_from_torch_axis = {
-      perm[i]: j for (i, j) in tensor_entry.returnn_axis_from_torch_axis.items()}
+      j: perm[i] for (i, j) in tensor_entry.returnn_axis_from_torch_axis.items()}
```

Comment:
This looks wrong.

Actually, the original code looks correct. Why do you think it is wrong?

Just the name `returnn_axis_from_torch_axis` implies that the keys are the Torch axes, while the values are the RETURNN axes. You are swapping keys and values here in your code, which would be wrong.

The `perm` should be applied on the Torch axes. I.e. on the keys. Which is what was done before.

Reply:
Hm, that's right. But I added the `test_movedim_2` to show that the current version fails in that case. I think then we need to take `perm` of `j` instead of `i`

```suggestion
      i: perm[j] for (i, j) in tensor_entry.returnn_axis_from_torch_axis.items()}
```

Reply:
This also works. I think it is even equivalent with my suggestion above (`i: perm[j]`)

Reply:
Ah yes I think you are right. This always confuses me.